UnitTests/CCUnitTestAssertions.h

```diff
@@ -1,19 +0,0 @@
-#define CCAssertEqualStrings(a1, a2, format...) XCTAssertEqualObjects(a1, a2, format)
```

Comment:
Was my fault in the first place, didn't recognize for a while that EqualObjects is doing string comparison just fine :)